README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center">
+<p style="text-align: center">
```

Comment:
I may be misremembering, but I think I remember someone making this update a while back and finding that's not supported — can you confirm?

Reply:
you can just edit in the browser to see. the `align` tag is not actually supposed to be part of html5. latest chrome looks like: 
![image](https://user-images.githubusercontent.com/3009477/146483244-db026970-3b93-4677-87fb-7ab40f39868a.png)

Reply:
(I only changed it because intellij complained "hey this is an invalid attribute" so i looked it up

Reply:
this is not really part of the PR, so I'll just revert it anyway

Reply:
@kzantow Oh yeah — I totally think `style=...` is better, I just thought there was a specific incompatibility w/ GitHub or something. I could be totally crazy :D